test/0.4.24/contracts/StakingRouter__MockForLidoAccounting.sol

```diff
@@ -47,4 +57,75 @@ contract StakingRouter__MockForLidoAccounting {
         totalFee__mocked = _totalFee;
         precisionPoint__mocked = _precisionPoints;
     }
+
+    function getStakingModuleIds() public view returns (uint256[] memory) {
+        return stakingModuleIds__mocked;
+    }
+
+    function getRecipients() public view returns (address[] memory) {
+        return recipients__mocked;
+    }
+
+    function getStakingModule(uint256 _stakingModuleId) public view returns (StakingRouter.StakingModule memory) {
+        if (_stakingModuleId >= 4) {
+            revert("Staking module does not exist");
+        }
+
+        if (_stakingModuleId == 1) {
```

Comment:
Maybe add a comment that production values are used? And if possible to add a proof link. Otherwise no one will guess why the numbers are like they are here.